app/src/main/java/de/fraunhofer/fokus/OpenMobileNetworkToolkit/QuickFragment.java

```diff
@@ -343,4 +344,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         updateUIHandler.postDelayed(updateUI, 500);
         return view;
     }
+
+    public String getSafeString(@StringRes int resId) {
+        if (context == null) {
+            return "NOT AVAILABLE AT THE MOMENT";
```

Comment:
Please replace the return value to "N/A"

Reply:
right. sorry about this slip up; I left it as this intentionally (for us) so that I can make a visual difference between when it "failed but prevented crash" and when it worked fine.

will do.

Reply:
done. resolving.